### PR TITLE
Many More Modifiers

### PR DESCRIPTION
**Pull request by Jack-Papel**

### CODE REVIEW

File: client/src/resources/lang/en_us.json [hunk 1]
```diff
@@ -1180,6 +1188,20 @@
     "wiki.article.standard.noDeathCause.text":"No Death Cause is a game modifier that changes the game such that every grave does not have a death cause listed.",
     "wiki.article.standard.roleSetGraveKillers.title":"Role Set Grave Killers",
     "wiki.article.standard.roleSetGraveKillers.text":"Role Set Grave Killers is a game modifier that changes the game such that every grave killer which is a role is replaced with one of:\n* Syndicate;\n* Town;\n* Minions;\n* Neutral;\n* Fiends; or\n* Cult.\n\nOr, if the killer's role is not in one of these role sets, it is not changed.",
+    "wiki.article.standard.noDueProcess.title": "No Due Process",
+    "wiki.article.standard.noDueProcess.text": "No Due Process is a game modifier that changes the game such that every nominated player will be executed without judgement.",
+    "wiki.article.standard.twoThirdsMajority.title": "Two-thirds Majority",
+    "wiki.article.standard.twoThirdsMajority.text": "Two-thirds Majority is a game modifier that changes the game such that during nomination and judgement, a two-thirds majority is required, rather than a simple majority.",
```
Comment (ItsSammyM) [inline client/src/resources/lang/en_us.json:1194]: two thirds rounded down or up?

File: client/src/resources/lang/en_us.json [hunk 2]
```diff
@@ -1180,6 +1188,20 @@
...
+    "wiki.article.standard.noDueProcess.text": "No Due Process is a game modifier that changes the game such that every nominated player will be executed without judgement.",
+    "wiki.article.standard.twoThirdsMajority.title": "Two-thirds Majority",
+    "wiki.article.standard.twoThirdsMajority.text": "Two-thirds Majority is a game modifier that changes the game such that during nomination and judgement, a two-thirds majority is required, rather than a simple majority.",
+    "wiki.article.standard.noTrial.title": "No Trial",
+    "wiki.article.standard.noTrial.text": "No Trial is a game modifier that changes the game such that the trial phases Nomination, Testimony, and Judgement no longer happen.",
+    "wiki.article.standard.noWhispers.title": "No Whispers",
+    "wiki.article.standard.noWhispers.text": "No Whispers is a game modifier that changes the game such that no players can whisper each other.",
+    "wiki.article.standard.noBlockMessages.title": "No Block Messages",
+    "wiki.article.standard.noBlockMessages.text": "No Block Messages is a game modifier that changes the game such that block messages cannot be sent.",
```
Comment (ItsSammyM) [inline client/src/resources/lang/en_us.json:1200]: What happens when you try? What happens when you go to send your alibi?

**Review by ItsSammyM — CHANGES_REQUESTED** [review]
Merge from main please, theres some problems

You should get a feedback message when a whisper message fails to send (you will see i did this when you go to merge from main)

Also I feel like I shouldn't approve a PR with modifiers that I strongly want to never ever use.

What could possibly be the purpose of no block messages? That is strictly making the UI worse and doing nothing else.

File: client/src/resources/lang/en_us.json (hunk 1 above)
Comment (Jack-Papel) [reply client/src/resources/lang/en_us.json:1194]: The smallest integer greater than or equal to two thirds

File: client/src/resources/lang/en_us.json (hunk 2 above)
Comment (Jack-Papel) [reply client/src/resources/lang/en_us.json:1200]: Button is greyed out

File: client/src/resources/lang/en_us.json (hunk 1 above)
Comment (Jack-Papel) [reply client/src/resources/lang/en_us.json:1197]: So yeah rounded up

Comment (ItsSammyM) [reply client/src/resources/lang/en_us.json:1213]: Please make that clear in the encyclopedia

File: client/src/resources/lang/en_us.json (hunk 2 above)
Comment (ItsSammyM) [reply client/src/resources/lang/en_us.json:1219]: It should say that. Id rather you remove this modifier altogether though

**Review by ItsSammyM — CHANGES_REQUESTED** [review]
I would rather the way you make it so they cant talk in chats is by removing them from the chat groups because then it would show no chat groups and there would be less potential edge cases for me to deal with.

Although the chat groups are a huge mess right now so that maybe can change later

I don't want to approve this until we have a conversation about the gameplay of these.

No chat, and no block messages specifically i have problems with.

**Review by ItsSammyM — APPROVED** [review]
Okay its a go

**Pull request merged**